homeassistant/components/media_player/sonos.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, hass, player):
         self.hass = hass
         self.volume_increment = 5
         self._player = player
+        self._name = None
```

Comment:
I personally hate that with the `Plex` media player - it causes my scripts
to swap on HASS start between living room and other rooms because I am
literally unable to change the plex client id on some of my TVs. I would
second using uuid as name - or even a concatenation of name and UUID

On Tue, Sep 6, 2016 at 2:14 AM, Valentin Alexeev <notifications@github.com>
wrote:

> In homeassistant/components/media_player/sonos.py
> <https://github.com/home-assistant/home-assistant/pull/3217#discussion_r77590182>
> :
>
> > @@ -199,6 +199,7 @@ def __init__(self, hass, player):
> >          self.hass = hass
> >          self.volume_increment = 5
> >          self._player = player
> > +        self._name = None
>
> @MartinHjelmare <https://github.com/MartinHjelmare> ok, now I understand
> it. Thank you for clarification! I rest my case :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/3217/files/d79b5960bb59b868fef176b0a915622c7f888af5#r77590182>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC2fZYB22Al4L_WwCbsqszIH9vXiB0FEks5qnSDegaJpZM4J1T2g>
> .
>